client/my-sites/people/invite-people/index.jsx

```diff
@@ -190,9 +190,10 @@ class InvitePeople extends React.Component {
 
 	async validateInvitation( siteId, usernamesOrEmails, role ) {
 		try {
-			const { success, errors } = await wpcom
-				.undocumented()
-				.createInviteValidation( siteId, usernamesOrEmails, role );
+			const { success, errors } = await wpcom.req.post( `/sites/${ siteId }/invites/validate`, {
```

Comment:
Noting that omitting the second empty object arg here works well. The `wpcom.req.post()` API is not so straightforward as it comes to handling a variable number of arguments, but in this case, when we have 2 args, it'll correctly transfer the query args to be body: https://github.com/Automattic/wp-calypso/blob/74a151a0434fa55e0f3b63b8382ef56c0e77ee80/packages/wpcom.js/src/lib/util/request.js#L44

Reply:
Yes, I looked at this source code to make sure the extra `{}` args can be removed.

`wpcom` also does a lot of unintuitive magic when seeing `apiVersion` or `apiNamespace` in the `query` argument. It detects that it's not really a query argument, and removes it from there. While still respecting it as if it was in the first `params` argument. I wish this magic wasn't there and that the library was more strict about its inputs.